README.md

```diff
@@ -1,2 +1,81 @@
 # TEST-REVIEW
 To try and review the addmission code
+>  project
+-This project is aimed at visiting a pass code and editing it back.
+- Make few necesary changes
+- Review the code your self then submit for further review
+
```

Comment:
That's a great project description 😄 , But don't you think it gives very little information about the project, some few additional details will do 👍🏾